### PR TITLE
Subscriptions widget styling updates

### PR DESCRIPTION
**Pull request by epwinchell**
Depends on: https://github.com/RedHatInsights/widget-layout/pull/97

[RHCLOUD-32808](https://issues.redhat.com/browse/RHCLOUD-32808)

Old
Screenshot 2024-06-24 at 9 49 02 AM

New
Screenshot 2024-06-24 at 9 48 03 AM

### CODE REVIEW

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

File: src/components/Widgets/SubscriptionsWidget.tsx [hunk 1]
```diff
@@ -3,20 +3,19 @@ import OutlinedCalendarAltIcon from '@patternfly/react-icons/dist/js/icons/outli
...
+import { EmptyState } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateVariant } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
+import { Stack } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { StackItem } from '@patternfly/react-core/dist/dynamic/layouts/Stack';
+import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import { Alert } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { AlertVariant } from '@patternfly/react-core/dist/dynamic/components/Alert';
+import { Gallery } from '@patternfly/react-core/dist/dynamic/layouts/Gallery';
```
Comment (jlgre) [inline src/components/Widgets/SubscriptionsWidget.tsx:15]: Question: what's the motivation for moving from the bulk import to these?

**Review by jlgre — COMMENTED** [review]
One small question

File: src/components/Widgets/SubscriptionsWidget.tsx (hunk 1 above)
Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: @jlgre I didn't do this. It happen automatically when I pushed the commit. I've not seen this before.

Comment (jlgre) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: Interesting - does it happen on push or on commit? I've never had that happen to my pull requests

Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: On push. I then immediately see that I need to rebase.

Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: I just committed changes restoring the imports, and the only thing that came through was a line break.

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

File: src/components/Widgets/SubscriptionsWidget.tsx (hunk 1 above)
Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: I could try putting it back manually on GH.

Comment (jlgre) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: Yeah, I would give that a try.

It probably isn't that important, but I would rather it not change

Comment (jlgre) [reply src/components/Widgets/SubscriptionsWidget.tsx:15]: https://github.com/RedHatInsights/subscription-inventory-ui/pull/186 - it looks like I'm able to make a change in that file and the imports not reorder.

Wonder if its cause its a fork? It shouldn't though

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

File: src/components/Widgets/SubscriptionsWidget.tsx (hunk 1 above)
Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx]: @jlgre Updated. I'll have another PR soon to update the Empty State styling. Thanks for your help.

Comment (jlgre) [reply src/components/Widgets/SubscriptionsWidget.tsx]: Sweet - what ended up to be causing this? Just curious cause I'd never seen it before

Comment (epwinchell) [reply src/components/Widgets/SubscriptionsWidget.tsx]: @jlgre No clue - 10 years on GH and I've never had this happen.

**Comment by InsightsDroid** [comment]
## :exclamation: Outdated dependencies error
* Installed version of @redhat-cloud-services/frontend-components-config (5.0.5) is older than required version (6.0.0)

**Review by jmbuscemi — APPROVED** [review]

**Pull request merged**